tools/tests.webpack.js

```diff
@@ -1,2 +1,9 @@
+var matchers = require('helpers/teaspoon-jasmine-matchers');
```

Comment:
Sure, because you define config for the webpack, that will define a loader, that will allow to use ES6.